### ISSUE TITLE
curl_exec failed calling URL #carousel #2

### ISSUE THREAD

**Issue by rodseb** [johnbe4/getSeoSitemap#2]
Hello I allow you to re-open a ticket view that the other you have closed
http://prntscr.com/hfu7lc

here is the code
https://hastebin.com/ayosenayet.js

**Comment by johnbe4** [comment]
Please just copy and paste html of the section that includes "#carousel #2 #2".

**Comment by rodseb** [comment]
[20171125.log](https://github.com/johnbe4/getSeoSitemap/files/1506403/20171125.log)

**Comment by johnbe4** [comment]
I do not need the log file.
Just copy and paste here the portion of code inside ayosenayet.js near ""#carousel".

**Comment by rodseb** [comment]
Hello
I'm coming back to you
hold the link
https://github.com/NeoFragCMS/neofrag-cms/blob/master/widgets/gallery/views/slider.tpl.php
https://hastebin.com/usetufegib.js

**Comment by johnbe4** [comment]
Hi,
Just copy and paste here the portion of html code inside ayosenayet.js near ""#carousel".

**Comment by rodseb** [comment]
```
/*https://github.com/noreiller/bootstrap-modal-carousel/blob/master/dist/js/bootstrap-modal-carousel.js*/
/*! bootstrap-modal-carousel 2014-03-07 */
(function(factory) {
	"use strict";
	if (typeof define === "function" && define.amd) {
		define([ "jquery", "bootstrap" ], factory);
	} else {
		factory(window.jQuery);
	}
})(function($) {
	"use strict";
	$.extend($.fn.carousel.Constructor.prototype, {
		fit: function(height) {
			if (!height) {
				height = $(".item.active", this.$element).height();
			}
			if (height != this.$element.height()) {
				if ($.support.transition) {
					this.$element.animate({
						height: height
					});
				} else {
					this.$element.height(height);
				}
				this.$element.trigger($.Event("fit.bs.carousel", {
					height: height
				}));
			}
			if (this.$element.find(".item.active img").length == 1) {
				var $img = this.$element.find(".item.active img"), pos = {
					left: $img.position().left,
					right: $img.position().left
				};
				if ($.support.transition) {
					this.$element.find(".carousel-caption").animate(pos);
				} else {
					this.$element.find(".carousel-caption").css(pos);
				}
			}
		}
	});
	$(document).on("slide.bs.carousel", ".carousel.carousel-fit", function(event) {
		var data = $(this).data("bs.carousel");
		data.$element.height($(".item.active", data.$element).height());
		var $next = $(".item.active", data.$element)[event.direction == "left" ? "next" : "prev"]();
		if (!$next.length) {
			$next = $(".item:" + (event.direction == "left" ? "first" : "last") + "-child", data.$element);
		}
		var height;
		height = $next.height();
		data.fit(height);
	}).on("slid.bs.carousel", ".carousel.carousel-fit", function(event) {
		var data = $(this).data("bs.carousel");
		data.fit();
	});
});

(function (factory) {
	'use strict';
	if (typeof define === 'function' && define.amd) {
		// Register as an anonymous AMD module:
		define([
			'jquery',
			'bootstrap'
		], factory);
	} else {
		// Browser globals:
		factory(
			window.jQuery
		);
	}
}(function ($) {
	'use strict';

	/**
	 * Update the carousel top margin
	 * @param  {object} $carousel The carousel element
	 * @param  {number} height    The height of the carousel
	 */
	function updateCarouselTopMargin ($carousel, height) {
		if (!height) {
			height = $carousel.height();
		}

		var
			$parent = $carousel.parents('.modal:first')
			, needHeadingHandle = !$parent.hasClass('force-fullscreen')
			, parentFreeSpace = $parent.height()
		;

		if (needHeadingHandle) {
			parentFreeSpace = parentFreeSpace - $('.modal-header', $parent).height();
			parentFreeSpace = parentFreeSpace - $('.modal-footer', $parent).height();
		}

		if ($.support.transition && $carousel.hasClass('slide')) {
			$carousel.animate({
				marginTop: (parentFreeSpace - height) / 2
			});
		}
		else {
			$carousel.css({
				marginTop: (parentFreeSpace - height) / 2
			});
		}
	}

	$(document)
		// When the modal is shown, fit the carousel
		.on('shown.bs.modal', '.modal', function (event) {
			if ($('.carousel', this).length) {
				$('.carousel', this).data('bs.carousel').fit();
			}
		})
		// When the fullscreen modal is shown, place the carousel at the center
		.on('shown.bs.modal', '.modal.modal-fullscreen', function (event) {
			if ($('.carousel', this).length) {
				updateCarouselTopMargin($('.carousel', this).data('bs.carousel').$element);
			}
		})
		// When the carousel is adjusted in a fullscreen modal, adjust its position
		.on('fit.bs.carousel', '.modal.modal-fullscreen .carousel', function (event) {
			updateCarouselTopMargin($(this).data('bs.carousel').$element, event.height);
		})
		// When a carousel comes back to its native postion, remove the top
		// margin and fit it if needed.
		.on('replaced.bs.local', '.carousel', function (event) {
			$(this).css('margin-top', 0);

			if ($(this).hasClass('carousel-fit')) {
				$(this).data('bs.carousel').fit();
			}
		})
	;
}));

(function(factory) {
	"use strict";
	if (typeof define === "function" && define.amd) {
		define([ "jquery", "bootstrap" ], factory);
	} else {
		factory(window.jQuery);
	}
})(function($) {
	"use strict";
	$.extend($.fn.modal.Constructor.prototype, {
		local: function(selector) {
			this.options.local = selector;
		},
		detach: function() {
			$(this.options.local).trigger($.Event("detach.bs.local"));
			if (!this.$placeholder) {
				this.$placeholder = $("<div></div>").addClass("hidden").html(this.$element.find(".modal-body").html()).insertBefore($(this.options.local));
			}
			this.$local = $(this.options.local).detach();
			this.$element.find(".modal-body").empty().append(this.$local);
			$(this.options.local).trigger($.Event("detached.bs.local"));
		},
		replace: function() {
			$(this.options.local).trigger($.Event("replace.bs.local"));
			this.$local.detach().insertAfter(this.$placeholder);
			this.$element.find(".modal-body").html(this.$placeholder.html());
			$(this.options.local).trigger($.Event("replaced.bs.local"));
		}
	});
	$(document).on("show.bs.modal", ".modal", function() {
		var data = $(this).data("bs.modal");
		if (data.options.local) {
			data.detach();
		}
	}).on("hidden.bs.modal", ".modal", function() {
		var data = $(this).data("bs.modal");
		if (data.options.local) {
			data.replace();
		}
	}).on("click.bs.modal.data-api", '[data-toggle="modal"][data-local]', function(event) {
		$($(this).attr("data-target")).modal("local", $(this).data("local"));
	});
});
```
it's the good one?
I am sorry
the CMS is not done by me

**Comment by johnbe4** [comment]
Hi,
I noticed there are those 2 href:s
- href="#carousel-example-generic"
- href="#carousel".
Please let me know more about those 2 hrefs.
Why are you using the char "#"?
Regards

**Comment by rodseb** [comment]
Hello
yes i know the href = "# carousel-example-generic" is original;
and the href = "# carousel". is the edit version

no function in verifier

**Comment by johnbe4** [comment]
Hi,
curl_exec does not accept url like  "#carousel".
It considers that url as malformed.
You should change the url using something like "carousel.htm".
Otherwise you should skip those urls using the var $skipUrl or $fileToAdd in my script.

**Comment by rodseb** [comment]
how to do it with $skipUrl
It does not work?

**Comment by johnbe4** [comment]
Hi,
Reading once again your log file,, getSeoSitemap tries to follow the url "#carousel".
Please let me know the absolute url of the page that contains:
 href="#carousel".
However, I'm almost ready with getSeoSitemap v2.0.
I will publish the new release here in the next days.
Please wait.

**Comment by rodseb** [comment]
thx @johnbe4

**Comment by johnbe4** [comment]
Hi,
I've just published getSeoSitemap v2.0.
Please note:
1 - you must delete getSeoSitemap table into your dBase before moving from v1.1 to v2.0.
2 - you must use absolute urls inside your web site to run getSeoSitemap without errors.
